CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## Unreleased
+
+### Language changes
+
+- Warn about deprecated constants
```

Comment:
Could you write a full sentence here please, mentioning the attribute.

Reply:
Done!